### DISCUSSION TITLE
Selection a session type/connection without using the login window

### DISCUSSION THREAD

**Discussion by CesiumCs** [neutrinolabs/xrdp#1935] [Q&A] [answered]
I have xrdp set up on one machine, and it has two connections: the default* xorg one, and a custom VNC one that connects to another machine on the local network. I'd like to be able to save my password in my rdp client so I don't have to type it in the gui every time I connect, but doing so makes it automatically proceed to the first session in xrdp.ini. What I was hoping to do was have two different connections in my client, and be able to have it connect to one session type or the other depending on what I chose in the client. Is there any way to do this?

**Comment by metalefty** [comment, marked as answer]
Domain can be used for such purpose. Try username like `Xvnc\username` or `Xorg/username`.

**Reply by CesiumCs** [reply]
Awesome! Thank you!

**Reply by metalefty** [reply]
There was a typo. The separator must be `\` (backslash).
